extensions/kusto/src/apiWrapper.ts

```diff
@@ -0,0 +1,116 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the Source EULA. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import * as vscode from 'vscode';
```

Comment:
Please remove this - it's unnecessary in my experience and just adds unneeded bloat.

Reply:
Removed it

Reply:
Seconded this